Meta/serenity.sh

```diff
@@ -191,14 +191,14 @@ pick_host_compiler() {
         return
     fi
 
-    find_newest_compiler clang clang-14 clang-15 /opt/homebrew/opt/llvm/bin/clang
+    find_newest_compiler clang clang-14 clang-15 clang-16 /opt/homebrew/opt/llvm/bin/clang
     if is_supported_compiler "$HOST_COMPILER"; then
         export CC="${HOST_COMPILER}"
         export CXX="${HOST_COMPILER/clang/clang++}"
         return
     fi
 
-    find_newest_compiler egcc gcc gcc-12 /usr/local/bin/gcc-12 /opt/homebrew/bin/gcc-12
+    find_newest_compiler egcc gcc gcc-12 gcc-13 /usr/local/bin/gcc-{12,13} /opt/homebrew/bin/gcc-{12,13}
```

Comment:
Does this kind of expansion work in the ancient bash macOS ships by default?

Reply:
It definitely does work under Homebrew bash, which is a build dependency.

I have just tested `/bin/bash` (version 3.2) now, and it works there as well.